packages/web-components/fast-foundation/src/data-grid/data-grid.ts

```diff
@@ -424,92 +438,65 @@ export class FASTDataGrid extends FASTElement {
                     break;
                 }
-                if (this.focusRowIndex === 0) {
-                    this.focusOnCell(0, this.focusColumnIndex, false);
-                    return;
-                }
+                console.log(this.getPageSize());
+                console.log(this.focusRowIndex);
+                console.log(this.focusRowIndex - this.getPageSize());
+                newFocusRowIndex = Math.max(0, this.focusRowIndex - this.getPageSize());
 
-                newFocusRowIndex = this.focusRowIndex - 1;
-
-                for (newFocusRowIndex; newFocusRowIndex >= 0; newFocusRowIndex--) {
-                    const thisRow: HTMLElement = this.rowElements[newFocusRowIndex];
-                    if (thisRow.offsetTop < this.scrollTop) {
-                        this.scrollTop =
-                            thisRow.offsetTop + thisRow.clientHeight - this.clientHeight;
-                        break;
-                    }
-                }
-
-                this.focusOnCell(newFocusRowIndex, this.focusColumnIndex, false);
+                this.focusOnCell(newFocusRowIndex, this.focusColumnIndex, "start");
                 break;
 
             case keyPageDown:
                 e.preventDefault();
                 if (this.rowElements.length === 0) {
-                    this.focusOnCell(0, 0, false);
+                    this.focusOnCell(0, 0, "nearest");
                     break;
                 }
 
-                // focus down one "page"
-                if (
-                    this.focusRowIndex >= maxIndex ||
-                    lastRow.offsetTop + lastRow.offsetHeight <= currentGridBottom
-                ) {
-                    this.focusOnCell(maxIndex, this.focusColumnIndex, false);
-                    return;
-                }
-
-                newFocusRowIndex = this.focusRowIndex + 1;
-
-                for (newFocusRowIndex; newFocusRowIndex <= maxIndex; newFocusRowIndex++) {
-                    const thisRow: HTMLElement = this.rowElements[
-                        newFocusRowIndex
-                    ] as HTMLElement;
-                    if (thisRow.offsetTop + thisRow.offsetHeight > currentGridBottom) {
-                        let stickyHeaderOffset: number = 0;
-                        if (
-                            this.generateHeader === GenerateHeaderOptions.sticky &&
-                            this.generatedHeader !== null
-                        ) {
-                            stickyHeaderOffset = this.generatedHeader.clientHeight;
-                        }
-                        this.scrollTop = thisRow.offsetTop - stickyHeaderOffset;
-                        break;
-                    }
-                }
-
-                this.focusOnCell(newFocusRowIndex, this.focusColumnIndex, false);
+                console.log(this.getPageSize());
+                console.log(this.focusRowIndex);
+                console.log(this.focusRowIndex + this.getPageSize());
```

Comment:
Nit, remove logs

Reply:
done

Reply:
@scomea, still seeing the console logs :)

Reply:
Oops.  Fixed.